app/models/container_group.rb

```diff
@@ -68,6 +69,14 @@ def event_where_clause(assoc = :ems_events)
     end
   end
 
+  def tenant_identity
```

Comment:
As @moolitayer said on https://github.com/ManageIQ/manageiq/pull/8815#diff-for-comment-65078382
it's unfortunate that this exact implementation already appeared 5 times (EmsCluster, ResourcePool, Host, OrchestrationStack, ContainerImage) and I'm adding 3 more.

There is no obvious mixin where it can go.
Since this PR is destined for backporting, and tenancy is complete black magic to me — I don't even know if I stole the right code, just followed what ContainerImage did — 
I plead to allow the duplication here, and have someone who understands tenancy refactor it separately.